packages/components/src/Chips/InternalChipDismissible/InternalChipDismissibleInput.tsx

```diff
@@ -66,10 +68,12 @@ export function InternalChipDismissibleInput(props: ChipDismissibleInputProps) {
     isInView && onLoadMore && onLoadMore(searchValue);
   }, [isInView]);
 
-  if (!menuOpen) {
-    return React.cloneElement(activator, { onClick: handleOpenMenu });
+  if (!showInput) {
+    return React.cloneElement(activator, { onClick: handleShowInput });
   }
```

Comment:
nice, even though I don't love what this component is doing to begin with - keeping it to a single return is great.